doc/man3/OSSL_CRMF_MSG_get0_tmpl.pod

```diff
@@ -66,6 +85,33 @@ of the given CertId I<cid>.
 OSSL_CRMF_CERTID_get0_issuer retrieves the issuer name
 of the given CertId I<cid>, which must be of ASN.1 type GEN_DIRNAME.
 
+OSSL_CRMF_ENCRYPTEDKEY_get1_encCert() decrypts the certificate in the given
+encryptedKey I<ecert>, using the private key I<pkey>, library context
+I<libctx> and property query string I<propq> (see L<OSSL_LIB_CTX(3)>).
+This is needed for the indirect POPO method as in RFC 4210 section 5.2.8.2.
+The function returns the decrypted certificate as a copy, leaving its ownership
+with the caller, who is responsible for freeing it.
+
+OSSL_CRMF_ENCRYPTEDKEY_get1_pkey() decrypts the private key in I<encryptedKey>.
+If `encryptedKey` is not of type B<OSSL_CRMF_ENCRYPTEDKEY_ENVELOPEDDATA>, 
+decryption uses the private key I<pkey>.
+The library context I<libctx> and property query I<propq> are taken into account as usual.
+The rest of this paragraph is relevant only if CMS support not disabled for the OpenSSL build
+and `encryptedKey` is of  type case B<OSSL_CRMF_ENCRYPTEDKEY_ENVELOPEDDATA>.
+Decryption uses the I<secret> parameter if not NULL; 
+otherwise uses the private key <pkey> and the certificate I<cert> 
+related to I<pkey>, where I<cert>  is recommended to be given if available. 
```

Comment:
nit: double spaces
```suggestion
and `encryptedKey` is of type case B<OSSL_CRMF_ENCRYPTEDKEY_ENVELOPEDDATA>.
Decryption uses the I<secret> parameter if not NULL; 
otherwise uses the private key <pkey> and the certificate I<cert> 
related to I<pkey>, where I<cert> is recommended to be given if available. 
```